src/main/java/com/exasol/hadoop/scriptclasses/HCatTableFiles.java

```diff
@@ -77,7 +77,7 @@ public static void run(ExaMetadata meta, ExaIterator iter) throws Exception {
             kerberosCredentials);
         
         // Check table type (must be managed or external, no view or index tables)
-        String tableType = tableMeta.getTableType().toUpperCase();
```

Comment:
Is there a reason why you removed the toUpperCase()? We cannot be sure that it is always uppercase, however we want to compare case insensitive. Probably we should change the following check
```if (!tableType.equals("MANAGED_TABLE") && !tableType.equals("EXTERNAL_TABLE") ) ```
to
```if (!tableType.equalsIgnoreCase("MANAGED_TABLE") && !tableType.equalsIgnoreCase("EXTERNAL_TABLE") ) ```